examples/c++-dataflow/run.rs

```diff
@@ -112,11 +113,59 @@ async fn main() -> eyre::Result<()> {
 
     let dataflow = Path::new("dataflow.yml").to_owned();
     build_package("dora-runtime").await?;
+    archive_node_bridge(root, build_dir).await?;
```

Comment:
Let's keep it like this but i'm really not a fan of having to do all this heavy lifting within the example...